modules/amf-registration/amf-registration-rest/amf-registration-rest-impl/src/main/java/com/liferay/amf/internal/resource/v1_0/AccountResourceImpl.java

```diff
@@ -1,18 +1,41 @@
 package com.liferay.amf.internal.resource.v1_0;
 
 import com.liferay.amf.resource.v1_0.AccountResource;
-import com.liferay.amf.model.Account;
+import com.liferay.amf.dto.v1_0.Account;
 
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.ServiceScope;
 
+import java.util.HashMap;
+import java.util.Map;
 /**
  * @author me
  */
 @Component(
 	properties = "OSGI-INF/liferay/rest/v1_0/account.properties",
 	scope = ServiceScope.PROTOTYPE, service = AccountResource.class
 )
-public class AccountResourceImpl extends BaseAccountResourceImpl {
+public class AccountResourceImpl extends BaseAccountResourceImpl   {
+
+	Map<Integer, Account> account = new HashMap<>();
+
+	public Account getAccount(Integer accountId) throws Exception {
+		return getAccount(accountId);
+	}
+
+	public void deleteAccount(Integer accountId) throws Exception {
+		deleteAccount(accountId);
+
+	}
+
+	public Account createAccount(Account account) throws Exception {
+
+		return createAccount(account);
+	}
+
+	public Account updateAccount(Integer accountId, Account account) throws Exception {
+
+		return updateAccount(accountId,account);
+	}
```

Comment:
you should call the `AccountLocalService` (the one you've created) and their CRUD methods respectively for each method defined in `AccountResourceImpl`. for that, you might need to include the `account-api` as a dependency in your `build.gradle` file. then, add something like this to call the service methods:

```
@Reference
private AccountLocalService _accountLocalService;
```

let me know if you have any other questions ;)